content/docs/06-integrations/00-aws-efs.md

```diff
@@ -0,0 +1,175 @@
+---
+title: 'AWS-EFS'
+metaTitle: 'AWS EFS Integration with Palette'
+metaDescription: 'AWS EFS storage add on into Spectro Cloud'
+hiddenFromNav: false
+isIntegration: true
+category: ['storage']
+logoUrl: 'https://registry.dev.spectrocloud.com/v1/csi-aws-efs/blobs/sha256:5d1eb98bb847489f341beda1407c14442854ab8e5910d0cc8da1a63636057927?type=image/png'
+
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+
+
+
+# AWS EFS
+
+Amazon Elastic File System (Amazon EFS) is a scalable file storage that allows for automatic encryption of data at rest and in transit. You can access information from an AWS EFS volume, within a specific region, no matter which availability zone. The cluster can be distributed across availability zones instead of having it in one location and replicating it across multiple times.
+
+Palette handles setting up the AWS EFS as a volume with ease when adding the persistentvolume storage container. Palette will dynamically provision the AWS EFS storage layer for the worker node. 
+
+## Usage
+
+There are two ways to add AWS EFS to Palette:
+
+1. Add EFS as a CSI layer in AWS/EKS.
+
+
+2. As an Add-on layer, which will create a new storage class using the AWS EFS file system.
+
+
+# Requirements
+
+- Create the Identity and Access Management (IAM) role that allows the driver to manage EFS access points.  See (EFSCSIControllerIAMPolicy)[https://aws.amazon.com/blogs/containers/introducing-efs-csi-dynamic-provisioning/]
+
+
+- Have a filesystem created and available before you provision AWS EFS to Palette.
+
+## Usage
```

Comment:
"usage" is repeated
in line number 24 same module exists. Can consider removing one